src/main/java/Duke.java

```diff
@@ -1,10 +1,23 @@
 public class Duke {
+
+    private static void showDivider() {
+        System.out.println("-----------------------------------------------------");
+    }
+
+    private static void showGreeting() {
```

Comment:
perhaps can create a Ui class to hold the things you want to print I.e greetings, divider etc

Reply:
Thank you for the suggestion! I will implement it in the future commit.